### PR TITLE
Feature/issue #55

### PR DESCRIPTION
**Pull request by amba178**
Edited User, took out email field will be provided by Devise. Updated spec file for user.

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -1,10 +1,7 @@
 class User < ActiveRecord::Base
    actable
-   validates:email,:email => true,
```
Comment (joaopapereira) [inline app/models/user.rb]: This should be taken care on Issue 60

File: app/models/job_seeker.rb [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  	def validate_year_of_birth
+
+  	 self.year_of_birth =~/\A\d{4}\z/ ? year=self.year_of_birth :
+  	         errors.add(:year_of_birth, "cannot be more than or less 4 digits")
+  	         
+  	  if year 
+  	  	 (year.to_i<=Date.today.year)&&( year.to_i>=(Date.today.year - YEARS)) ? 
+  	  	         self.year_of_birth = year.to_s : errors.add(:year_of_birth, 
+  	  	         	 "should be between #{Date.today.year}-#{Date.today.year-YEARS}")
+  	  end
```
Comment (patmbolger) [inline app/models/job_seeker.rb]: Look at using rails extensions for more readable code.  For instance you could say things like:

year_of_birth.between?(18.years.ago, 100.years.ago)

etc....

Also, I would recommend setting the minimum age at 18 (as in the code snippet above).  Actually, I would not bother with checking for max year (100).  You could just check on age 18 or older, and, if not, have the message say 'must be 18 years or older'.

File: app/models/user.rb [hunk 3]
```diff
@@ -1,10 +1,7 @@
...
-            :uniqueness => true,
-            :presence => true
-   validates_presence_of :first_name
-   validates_presence_of :last_name
-   validates   :phone, :phone => true
+   validates_presence_of :first_name, :last_name, :phone 
+   validates_format_of :phone, with:  /\A\(?([0-9]{3})\)?[-|" ".*]?([0-9]{3})[-|" ".*]?([0-9]{4})\z/,
+          :message => "phone number should be 619-234-7890 format"
```
Comment (patmbolger) [inline app/models/user.rb]: Check if you could use the validator that already exists at app/validators/phone_validator.rb

File: db/migrate/20151022163836_create_users.rb [hunk 4]
```diff
@@ -1,12 +1,12 @@
 class CreateUsers < ActiveRecord::Migration
   def change
     create_table :users do |t|
-      t.string :email
```
Comment (joaopapereira) [inline db/migrate/20151022163836_create_users.rb]: This change is not in this Feature

File: spec/models/user_spec.rb [hunk 5]
```diff
@@ -6,7 +6,6 @@
    
    describe 'Database schema' do
     it { is_expected.to have_db_column :id }
-    it { is_expected.to have_db_column :email}
```
Comment (joaopapereira) [inline spec/models/user_spec.rb]: Same as before

File: spec/models/user_spec.rb [hunk 6]
```diff
@@ -16,16 +15,6 @@
 
    describe 'check model restrictions' do 
 
-     describe 'Email check' do
```
Comment (joaopapereira) [inline spec/models/user_spec.rb]: Same as before

**Comment by joaopapereira** [comment]
Please take a look at these comments and push again

**Pull request closed without merging**